### DISCUSSION TITLE
BT-509 - Organisation eDelivery Gateway - Validation according to URL too restricted?

### DISCUSSION THREAD

**Discussion by gabrieledall** [OP-TED/eForms-SDK#294] [Q&A] [answered]
Regarding the valiation rules for BT-509 - Organisation eDelivery Gateway I wonder if validation according to URL-Parameters is correct:

According to [PEPPOL documentation](https://docs.peppol.eu/pracc/syntax/SearchNoticeResponse/rim-RegistryObjectList/rim-RegistryObject/rim-Slot@name=BuyerInformation/rim-Slot@name=BuyerElectronicAddress/), this field has been interpreted as source for adding a BuyerElectronicAddress to any eForms document. 

But the BuyerElectronicAddress type is defined as EAS - which is NOT (only?) a URL.

Will this validation be changed in one of the next SDKs?

**Comment by ferrakaop** [comment, marked as answer]
The eForms regulation defines BT-509 as a URL and it's described as "The organisation's uniform resource locator for exchange of data and documents." so validation against the URL pattern is appropriate.

EAS on PEPPOL seems to be a broader type of identifier (like VAT numbers) that addresses a different need https://docs.peppol.eu/pracc/codelist/EAS/
